topic/timeseries/exploratory_data_analysis.ipynb

```diff
@@ -102,12 +102,37 @@
     "engine = sa.create_engine(CONNECTION_STRING, echo=os.environ.get('DEBUG'))"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "First, import data into CrateDB. This is a shorthand notation for the same code\n",
+    "illustrated in `timeseries-queries-and-visualization.ipynb`, running corresponding\n",
+    "SQL DDL and DML statements, to load the data."
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "from cratedb_toolkit.datasets import load_dataset\n",
+    "\n",
+    "dataset = load_dataset(\"tutorial/weather-basic\")\n",
+    "dataset.to_cratedb(dburi=CONNECTION_STRING, table=\"weather_data\")"
```

Comment:
That significantly reduces the boilerplate code mentioned on a different comment, using improvements from two other patches.

- https://github.com/crate/cratedb-datasets/pull/9
- https://github.com/crate-workbench/cratedb-toolkit/pull/123